pkg/nsx/services/subnet/subnet.go

```diff
@@ -459,3 +461,22 @@ func (service *SubnetService) UpdateSubnetSet(ns string, vpcSubnets []*model.Vpc
 	}
 	return nil
 }
+
+// GetSharedSubnetPath generates Subnet Path for shared Subnet
+func (service *SubnetService) GetSharedSubnetPath(subnet *v1alpha1.Subnet) (string, error) {
+	anno := subnet.GetAnnotations()
+	associatedResource, ok := anno[common.AnnotationAssociatedResource]
+	if !ok {
+		return "", fmt.Errorf("no associated resource annotation in Shared Subnet %s/%s", subnet.Namespace, subnet.Name)
+	}
+	// associatedResource has the format projectID:vpcID:subnetID
+	parts := strings.Split(associatedResource, ":")
+	if len(parts) != 3 {
+		return "", fmt.Errorf("failed to parse associated resource annotation %s", associatedResource)
+	}
+	vpcInfoList := service.vpcService.ListVPCInfo(subnet.Namespace)
```

Comment:
here we use this just to get the orgID, which we only supports "default" here, I feel we can just use "default"

Reply:
Updated, thanks!